toolkits/global/packages/global-popup/scss/10-settings/_global-popup.scss

```diff
@@ -0,0 +1,6 @@
+$c-popup--padding-vertical: spacing(16);
```

Comment:
These variables feel a bit odd to me.
Vertical is generic for top and bottom, but left and right are different.
Would `$c-popup--padding: spacing(16, 32, 16, 24)` make more sense?

I know it's used later on to determine the position of the icon, but one of them is divided by two anyway so it's not manipulating the number. Maybe vars like `$c-popup--close-top` would make it clearer?